src/ui/linkformview.js

```diff
@@ -61,6 +82,35 @@ export default class LinkFormView extends View {
 		 */
 		this.unlinkButtonView = this._createButton( t( 'Unlink' ), 'unlink' );
 
+		/**
+		 * A collection of views which can be focused in the form.
+		 *
+		 * @readonly
+		 * @protected
+		 * @member {module:ui/viewcollection~ViewCollection}
+		 */
+		this._focusables = new ViewCollection();
+
+		/**
+		 * Helps cycling over {@link #_focusables} in the form.
+		 *
+		 * @readonly
+		 * @protected
+		 * @member {module:ui/focuscycler~FocusCycler}
+		 */
+		this._focusCycler = new FocusCycler( {
+			focusables: this._focusables,
+			focusTracker: this.focusTracker,
+			keystrokeHandler: this.keystrokes,
+			actions: {
+				// Navigate form fields backwards using the shift + tab keystroke.
+				focusPrevious: 'shift + tab',
```

Comment:
Couldn't we have a preset for it?... It will repeat in every form. Or, I'd rather go with a `FormController` which wraps `FocusCycler`. With time it may need to handle much more than `FocusCycler` currently.

In other words – FocusCycler will be a simple util, while FormController a more specialised one. Ofc, we can work on this later, but, in fact, we have another form already in the Image feature.